### DISCUSSION TITLE
NMEA Dataset

### DISCUSSION THREAD

**Discussion by FayeWong98** [Petrichor-Labs/nmea_data_convert#2] [Q&A] [answered]
Is there an open source NMEA dataset?

**Comment by nabelekt** [comment, marked as answer]
Hi, @wanghaiyang01. You can take a look at the test files in this repository under https://github.com/Petrichor-Labs/nmea_data_convert/tree/master/test_data. For example: 
- https://github.com/Petrichor-Labs/nmea_data_convert/blob/master/test_data/test_data_4.nmea
- https://github.com/Petrichor-Labs/nmea_data_convert/blob/master/test_data/test_data_5.nmea

You can also generate your own NMEA data using an Android phone and an app like NMEA Tools (https://play.google.com/store/apps/details?id=com.peterhohsy.nmeatools&hl=en_US&gl=US).

I do not know of other data sets. Google is probably your best bet.

All the best.
Thomas
